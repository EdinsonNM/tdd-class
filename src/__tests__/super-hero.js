import { getFlyingSuperHeros } from "../super-heros";
test("shoould return fly heors with toMatchSnapshot", () => {
	const flyHeros = getFlyingSuperHeros();
	expect(flyHeros).toMatchSnapshot();
});
test("shoould return fly heors with toMatchInlineSnapshot", () => {
	const flyHeros = getFlyingSuperHeros();
	expect(flyHeros).toMatchInlineSnapshot(`
		Array [
		  Object {
		    "name": "Dynaguy",
		    "powers": Array [
		      "disintegration ray",
		      "fly",
		    ],
		  },
		  Object {
		    "name": "Apogee",
		    "powers": Array [
		      "gravity control",
		      "fly",
		    ],
		  },
		]
	`);
});
