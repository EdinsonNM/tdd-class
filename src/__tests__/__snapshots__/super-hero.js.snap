// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shoould return fly heors with toMatchSnapshot 1`] = `
Array [
  Object {
    "name": "Dynaguy",
    "powers": Array [
      "disintegration ray",
      "fly",
    ],
  },
  Object {
    "name": "Apogee",
    "powers": Array [
      "gravity control",
      "fly",
    ],
  },
]
`;
